sklearn_pandas/dataframe_mapper.py

```diff
@@ -307,8 +327,11 @@ def _transform(self, X, y=None, do_fit=False):
             extracted.append(_handle_feature(Xt))
 
             alias = options.get('alias')
+            prefix = options.get('prefix')
```

Comment:
Perhaps we can put the default prefix here: `prefix = options.get('prefix', '')`. Same with suffix. So we don't need the `prefix or ''` above.

Reply:
make sense